internal/client.go

```diff
@@ -213,61 +224,113 @@ func (client *Client) setupUI() {
 	client.app.EnableMouse(true)
 }
 
-func (client *Client) shortenPlotId(id string) string {
+func shortenPlotId(id string) string {
 	if len(id) < 20 {
 		return ""
 	}
 	return fmt.Sprintf("%s...%s", id[:10], id[len(id)-10:])
 }
 
+// Active plots
+
+type activePlotsData struct {
+	host      string
+	plotId    string
+	status    int
+	phase     int
+	progress  int
+	startTime time.Time
+	plotDir   string
+	destDir   string
+}
+
+var activePlotsHeaders = []string{
```

Comment:
It's possible to reduce a lot of the boilerplate code by using reflection in `SortedTable`, but I wanted to keep things simple for now.